rel/relx.config.src

```diff
@@ -9,6 +9,7 @@ Filter = ['rabbitmq_codegen'
           ,'.settings'
           ,'skel'
           ,'parse_trans'
+          ,'sup'
```

Comment:
Because:
$ make && make build-release
===> Failed to solve release:
 Dependency sup is specified as a dependency but is not reachable by the system.